ndohyep/app/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
[Tests seem to be missing?](http://ways-of-working.readthedocs.org/en/latest/process.html#our-development-process) In the mean time I'll get this repository setup with Travis & Coveralls.